beast/observationmodel/observations.py

```diff
@@ -160,6 +160,7 @@ def gen_SimObs_from_sedgrid(
     compl_filter="F475W",
     ranseed=None,
     vega_fname=None,
+    use_weight='weight',
```

Comment:
Not a fan of the name of this keyword.  It sounds like usually the weight is not used unless you set this keyword.  Maybe `weight_to_use`?

Reply:
Putting this keyword in the category of things that make more sense in my brain than others' brains :).  `weight_to_use` sounds reasonable to me.